docs/rfcs/0007-export-direction.md

```diff
@@ -0,0 +1,221 @@
+  // "otelcol.Consumer" instead of an "output" block
+  input = otelcol.receiver.prometheus.default.output
+  client {
+    endpoint = env("OTLP_ENDPOINT")
+  }
+}
+```
+
+## Potential approaches
+
+### Approach 1 - Use exports as both inputs and outputs (current status quo)
+With this approach the component's exports could be either its inputs or its outputs. The writer of each component uses their best judgement on whether having the exports as inputs or outputs would be most appropriate.
+
+#### Pros
+- Maximum flexibility.
+- Prometheus uses a pull model. Otel uses a push model. Supporting both models in Flow could make it easier to express Prometheus and Otel logic. 
+#### Cons
+- Lack of consistency between components can be confusing.
+- Having components in the same config file using different paradigms looks confusing.
+
+### Approach 2 - Use exports as inputs only
+With this approach the component's exports are its inputs. Each component will have to specify to which other components its output data is going to.
+
+#### Pros
+
+#### Cons
+- It may be awkward to modify components which are already configured, so that they send data to more downstream components.
+
+### Approach 3 - Use exports as outputs only
+With this approach the component's exports are its outputs. Other components will feed those outputs to their inputs. Each component will have to specify from which other components its input data is coming from.
+
+#### Pros
+- More intuitive. Newcomers to Flow will browse the documentation and look for inputs and outputs of each component. They will likely assume that the non-exported arguments are the component's inputs, whereas the exported ones are the component's outputs.
+
+#### Cons
+
+## Other considerations
+
+### Which approach would be less verbose to write?
+
+Depending on the situation, either approach could be more verbose. For example:
+
+- A single Prometheus remote write being referenced by multiple scrapers is less verbose when the remote write specifies its inputs in the remote write itself (as an input). This is example 1.2 above.
+- A single scraper referencing multiple remote writes would be less verbose if the remote writes are specified in the scraper itself (as an output).
+
+Hence, nether option can be considered less verbose in the general case.
+
+### Ease of reading
+
+Ideally, one should be able to read a River config file from top to bottom easily. In addition:
+* It should be easy to see where a component's inputs are coming from.
+* It should be easy to see where a component's outputs are going to.
+
+It is not possible to achieve both in practice, just by reading a config file. That's because we only specify either where data came from, or where it's going to. It would be redundant to specify this in both the source component and the destination component. The only way to achieve both of those things is to use a UI tool for visualizing the graph.
```

Comment:
The UI currently does _not_ display data flows, but graph dependencies. Would the proposed change help with that?